irr/src/CMakeLists.txt

```diff
@@ -152,8 +152,10 @@ else()
 	endif()
 endif()
 
-if(ENABLE_OPENGL)
-	add_definitions(-D_IRR_COMPILE_WITH_OPENGL_)
+if(ENABLE_OPENGL OR (ENABLE_OPENGL3 AND NOT USE_SDL2))
+	if(ENABLE_OPENGL)
+		add_definitions(-D_IRR_COMPILE_WITH_OPENGL_)
+	endif()
 	set(OPENGL_DIRECT_LINK TRUE) # driver relies on this
```

Comment:
should be inside `if(ENABLE_OPENGL)`

Reply:
Hmm. In my testing, this made no difference. Why exactly is it needed, and for which code?

Reply:
the legacy GL driver needs it, but not the GL3 driver.